README.md

```diff
@@ -13,33 +14,33 @@ Please note that the code, and especially the building and running story, is in
 a **pre-alpha** state. Please take a look, but don't be surprised if you hit
 bugs or missing pieces.
 
-## Building
 
-### In-place
+## Getting started
 
-To build the binaries in-place,
+To launch Scope inside of a Docker container,
 
 ```
-make build
+sudo wget -O /usr/local/bin/scope \
+  https://github.com/weaveworks/scope/releases/download/latest_release/scope
+sudo chmod a+x /usr/local/bin/scope
+sudo scope launch
 ```
 
-Note that this doesn't build or include the latest version of the user
-interface. The UI is decoupled, living in `client` and following a node/gulp
-workflow. To build that and include it in the application binary,
+Now, open your web browser to **http://localhost:4040**. If you're using
+boot2docker, replace localhost with the output of **boot2docker ip**.
 
-```
-make client
-make static
-make build
-```
 
-Or, as a shortcut,
+## Build
+
+### Build in-place
```

Comment:
Should we include this?  We want to push everyone towards running on docker.  The fact that you guys develop on macs and want to run in place is just going to be confusing.